Vic2ToHoI4/Data_Files/blankMod/output/interface/culture_ideas.gfx

```diff
@@ -1,2423 +1,1870 @@
 spriteTypes = {
 	spriteType = {
 		name = "GFX_idea_culture_aborigine"
-		texturefile = "gfx/interface/culture/culture_aborigine.dds"
+		texturefile = gfx/interface/culture/culture_aborigine.dds"
```

Comment:
The starting quote got removed and should be put back.

Reply:
Solved